frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -179,8 +172,7 @@ class WorkspaceAdvanced extends React.Component {
 
   componentDidUpdate (prevProps, prevState) {
     const { state } = this
-    // console.log('%c<WorkspaceAdvanced> did update', `color: ${state.config.hexcolor}`, prevState, state)
-
+    console.log('%c<WorkspaceAdvanced> did update', `color: ${state.config.hexcolor}`, prevState, state)
```

Comment:
the DidUpdate event spam a bit too much, so I think it's better to avoid avoid logging it everytime

Reply:
Changed